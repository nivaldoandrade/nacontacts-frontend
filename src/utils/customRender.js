/* eslint-disable import/no-extraneous-dependencies */
import { render } from '@testing-library/react';
import { ThemeProvider } from 'styled-components';

import theme from '../assets/styles/themes/default';

export const customRender = (ui, options) =>
  render(<ThemeProvider theme={theme}>{ui}</ThemeProvider>, options);
