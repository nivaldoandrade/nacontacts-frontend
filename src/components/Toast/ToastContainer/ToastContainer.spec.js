import { ToastContainer } from '.';
import { customRender } from '../../../utils/customRender';

describe('ToastContainer Component', () => {
  it('renders correctly', () => {
    const { container } = customRender(<ToastContainer />);
    expect(container).toMatchSnapshot();
  });
});
