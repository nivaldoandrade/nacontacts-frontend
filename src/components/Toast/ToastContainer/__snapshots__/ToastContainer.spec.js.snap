// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToastContainer Component renders correctly 1`] = `
<div>
  <div
    class="sc-gswNZR hyXmLs"
  />
</div>
`;
